src/test/java/com/exasol/AbstractLuaVirtualSchemaIT.java

```diff
@@ -63,9 +63,12 @@ private static boolean isLua54Used() {
         return !DOCKER_DB.contains(":7.0");
     }
 
-    protected VirtualSchema createVirtualSchema(final Schema sourceSchema) {
+    protected VirtualSchema createVirtualSchema(final Schema sourceSchema, final Map<String, String> properties) {
         final String name = sourceSchema.getName();
         final AdapterScript adapterScript;
+        final Map<String, String> mergedProperties = new HashMap<>();
+        mergedProperties.putAll(DEBUG_PROPERTIES);
```

Comment:
Does it work fine with the CI build? I thought when netcat wasn't listening the virtual schemas had problems

Reply:
Removed this because it was for local debugging and I simply forgot that this was local.